test/spriteBackgroundImages.js

```diff
@@ -556,4 +556,51 @@ describe('spriteBackgroundImages', () => {
       /are you trying to add an SVG to a sprite/
     );
   });
+
+  describe('with padding', function () {
+    describe('defined in the group selector', function () {
+      it('should apply the padding', async () => {
+        const assetGraph = new AssetGraph({
+          root: pathModule.resolve(
+            __dirname,
+            '..',
+            'testdata',
+            'spriteBackgroundImages',
+            'padding',
+            'inGroupSelector'
+          ),
+        });
+        await assetGraph.loadAssets('style.css');
+        await assetGraph.populate();
+
+        await assetGraph.queue(spriteBackgroundImages());
+        const [spriteAsset] = assetGraph.findAssets({ fileName: 'sprite.png' });
+        await expect(spriteAsset.rawSrc, 'to have metadata satisfying', {
+          size: {
+            width: 12,
+            height: 90,
```

Comment:
I'm a bit surprised about height 90. But I guess that would be padding behavior. I was more expecting a margin collapsing behavior, resulting in `50` being the result here.

I would expect if I set the padding property, I get that amount of empty space around each sprite. But I don't need double that. Since empty space is equal no matter if I'm looking at it from the perspective of one sprite or the other it should be fine. But I can see how that might be difficult to express for a sprite packer

Reply:
Yeah, I was also a bit :thinking: about it, but this is not really changing how the packers work, it just adds another way to apply it to a set of images.

I think I remember that the "Jim Scott" packer doesn't/can't easily support padding, but the horizontal and vertical packers should be easy to get to work the way you describe with collapsing of padding etc.

Reply:
@Munter, should we get this in and revisit the packer behavior separately?

Reply:
Yeah. This is better than before. Lets merge